src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Validator/Constraints/ValidAttribute.php

```diff
@@ -31,11 +32,18 @@ class ValidAttribute extends Constraint
 
     public $messageAttributeMotivationNotSet = 'validator.attribute.motivation_not_set';
 
+    public $type = [Constants::TYPE_SAML, Constants::TYPE_OPENID_CONNECT_TNG];
```

Comment:
These seem to be the allowed values for the type field, is that correct? 

If that's the case. Would these values not be overwritten when setting the `type` on the command? I'd expect these values to be used to verify we are setting a valid value for `type`.